src/lib/viewers/BaseViewer.js

```diff
@@ -724,48 +739,36 @@ class BaseViewer extends EventEmitter {
     }
 
     /**
-     * Shows the point annotate button.
+     * Shows the annotate button for the specified mode
      *
-     * @param {Function} handler - Point annotation button handler
+     * @param {string} currentMode - Annotation mode
      * @return {void}
      */
-    showPointAnnotateButton(handler) {
-        if (!this.isAnnotatable('point')) {
-            return;
-        }
+    showModeAnnotationButton(currentMode) {
+        const modes = {
```

Comment:
Move this out to a higher level constant. Doesn't need to be created every time we call this method